microprofile/WFLY-13089-microprofile-rest-client-1.4.adoc

```diff
@@ -0,0 +1,54 @@
+= EAP7-1429 / WFLY-13089 Microprofile REST Client 1.4 support
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+This is about providing an implementation of the Microprofile REST Client 1.4 spec (https://github.com/eclipse/microprofile-rest-client/releases/tag/1.4.0)
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/WFLY-13089[WFLY-13089]
+* {issue-base-url}/EAP7-1429[EAP7-1429]
+
+=== Related Issues:
+
+* {issue-base-url}/RESTEASY-2481[RESTEASY-2481]
+
+=== Dev Contacts:
+
+* mailto:rsigal@redhat.com[Ron Sigal]
+* mailto:asoldano@redhat.com[Alessio Soldano]
+
+=== QE Contacts:
+
+* mailto:jkasik@redhat.com[Jan Kasik]
+
+=== Affected Projects or Components:
+
+* WildFly
+* RESTEasy
+
+== Requirements
+
+* RESTEasy provides an implementation of Microprofile REST Client 1.4.0 version
+** The only relevant change comparing to version 1.3.x is the requirement to support CDI injection in ClientHeadersFactory class
```

Comment:
What are the criteria for a "relevant change"? Is this the only change affecting user of the API?

Reply:
Right, changes impacting the user applications or requiring a modification of the spec implementation.